frontend/mobilityai/app/src/main/java/com/ai/mobility/mobilityai/BtleScanner.java

```diff
@@ -0,0 +1,68 @@
+package com.ai.mobility.mobilityai;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+public class BtleScanner extends Fragment {
+
+    private OnFragmentInteractionListener mListener;
+
+    public BtleScanner() {
+        // Required empty public constructor
+    }
+
+    public static BtleScanner newInstance(String param1, String param2) {
+        BtleScanner fragment = new BtleScanner();
+        Bundle args = new Bundle();
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_btle_scanner, container, false);
+    }
+
+    // TODO: Rename method, update argument and hook method into UI event
```

Comment:
Should we create an issue for this TODO so we don't forget?

Reply:
Done, see #78